### DISCUSSION TITLE
Disable Android updates specific apps

### DISCUSSION THREAD

**Discussion by ToToGSARider** [devemux86/cruiser#642] [Q&A] [answered]
If I deactivate the update in the Cruiser app in the Playstore, an update is still suggested to me. What happens if I report the app as inappropriate? Will it then be temporarily removed from my update list for the device? Can I then mark the app as appropriate on this device again later? I don't want Cruiser 5 to be installed on my navigation phone as long as 3D is not available. 

Wenn ich im Playstore das Update deaktiviere in der Cruiser App, wird mir trotzdem ein Update vorgeschlagen. Was passiert, wenn ich die App als unangemessen melde? Wird sie dann vorübergehend aus meiner Updateliste bei dem Gerät entfernt? Kann ich dann später die App wieder als angemessen auf diesem Gerät markieren? Ich möchte ja nicht, dass auf meinem Navigationshandy die Cruiser 5 installiert wird, solange 3D nicht vorhanden ist.

**Comment by devemux86** [comment, marked as answer]
How to disable individual Android apps from receiving automatic updates:

https://www.phonearena.com/news/Disable-Android-updates-specific-apps_id102881
